composer.json

```diff
@@ -58,10 +58,7 @@
             "locations": {
                 "web-root": "./build"
             }
-        },
-        "_readme": [
```

Comment:
I removed this note as its outdated, now we list them as dependency of the submodules